nix/un-pearl.nix

```diff
@@ -0,0 +1,56 @@
+# This module attempts to be a workaround to
+# remove the dependency of pearl from the
+# crictical path of using kexec images
+# in combination with nixos-anywhere
+
+# THIS IS NOT FUNCTIONAL
```

Comment:
We also have sysusers integrated in NixOS, which is already part of systemd.

Reply:
Ah interesting! I did see it briefly without grasping its purpose or function, at all.

You're saying: it's a substitute to the `update-users-groups.pl` which is _awesome_, because that file is a lot more complex to rewrite in `c`.

Nice!